src/main/java/org/lmdbjava/Dbi.java

```diff
@@ -50,7 +50,7 @@
   private final String name;
   private final Pointer ptr;
 
-  Dbi(final Env<T> env, final Txn<T> txn, final String name,
+  Dbi(final Env<T> env, final Txn<?> txn, final String name,
```

Comment:
The type parameter of the transaction is irrelevant here, we just need the pointer.